Directory.Build.targets

```diff
@@ -17,30 +17,43 @@
     Microsoft.WorkflowBuildExtensions.targets target that always runs in VS
     -->
     <GenerateCompiledExpressionsTempFilePathForEditing></GenerateCompiledExpressionsTempFilePathForEditing>
+
+    <!-- Pass RuntimeMetadataVersion to the compiler for all Core Assemblies -->
+    <RuntimeMetadataVersion Condition="'$(IsCoreAssembly)' == 'true'">v4.0.30319</RuntimeMetadataVersion>
+
+    <CLSCompliant Condition="'$(CLSCompliant)'==''">true</CLSCompliant>
   </PropertyGroup>
   
   <!--
   Hack workaround for not restoring each project. Instead, we turn off all the targets
   that require a `project.assets.json`, since ours will be empty anyway.
   -->
   <Target Name="_ClearResolvePackageAssets"
-          BeforeTargets="AssignProjectConfiguration">
+          BeforeTargets="AssignProjectConfiguration"
+          Condition="'$(MSBuildProjectExtension)' != '.depproj'">
      <PropertyGroup>
       <ResolvePackageDependenciesForBuildDependsOn></ResolvePackageDependenciesForBuildDependsOn>
     </PropertyGroup>
   </Target>
 
-  <Target Name="CheckForBuildTools">
-    <Error Condition="!Exists('$(ToolsDir)') and '$(OverrideToolsDir)'=='true'"
-           Text="The tools directory [$(ToolsDir)] does not exist. Please run sync in your enlistment to ensure the tools are installed before attempting to build an individual project." />
-    <Error Condition="!Exists('$(ToolsDir)') and '$(OverrideToolsDir)'!='true'"
-           Text="The tools directory [$(ToolsDir)] does not exist. Please run init-tools.cmd in your enlistment to ensure the tools are installed before attempting to build an individual project." />
+  <!--
+  Hack workaround for .depproj's built with TargetFramework=net46
+  This causes us to unnecessarily attempt to restore framework assemblies from the project directory
+  This target clears out those assemblies from the list of assets to resolve -->
+  <Target Name="_ClearResolvedFrameworkAssemblies"
```

Comment:
I currently see 3 depproj files in the standard repo. Which one has a configuration for net46?

Reply:
Ah, the comment should say "net47": https://github.com/dotnet/standard/blob/master/apicompat/netstandard1x/netstandard.builds#L29-L31